fiftyone/operators/executor.py

```diff
@@ -471,16 +472,20 @@ def view(self):
         if self._view is not None:
             return self._view
 
-        # Forces a dataset reload if necessary
-        _ = self.dataset
+        # Always derive the view from the context's dataset
+        dataset = self.dataset
+        if not dataset:
```

Comment:
This error check is unnecessary: if a valid dataset could not be loaded an error would have already been raised when `self.dataset` is called.